docs/source/time-correlation/time-correlation.rst

```diff
@@ -153,19 +164,29 @@ A common trick
 ************************************
 | Numerical simulations providing the dynamics over time of the dipole (as well as any other quantity) can be used to evaluate the previous formulas only if the simulat time is long enough and the dynamics is properly captured, i.e. if the integration time step is small enough.
 | In certain systems, like liquid water, properly sampling the long time behavior of :math:`\left(\boldsymbol{\mu}\star\boldsymbol{\mu}\right)\left(t\right)`, i.e. the low frequency behavior of its Fourier transform, can be challenging due to its intrisically "low dynamics".
-| This means that to properly sample the low frequrncy region of :math:`S\left(\omega\right)`, extremely long simulations would be needed.
+| This means that to properly sample the low frequency region of :math:`\mathcal{I}\left(\omega\right)`, extremely long simulations would be needed.
 | There is however a possible, or partial, solution that can be applied to the infrared spectrum, which consists in relating :math:`\left(\dot{\boldsymbol{\mu}}\star\dot{\boldsymbol{\mu}}\right)\left(t\right)` with :math:`\boldsymbol{\mu}\star\boldsymbol{\mu}(t)`, i.e. the TCF of the dipole time derivative.
 | It is not hard to show that:
 
 .. math::
     \omega^2 \mathcal{F}\left[\boldsymbol{\mu}\star\boldsymbol{\mu}\right]\left(\omega\right) = 
     \mathcal{F}\left[\dot{\boldsymbol{\mu}}\star\dot{\boldsymbol{\mu}}\right]\left(\omega\right)
 
+.. attention::
+    Just for completeness, it is worth mentioning that this relation is valid only in the continuous case, while in the discrete case some care should be taken when evaluating the derivative of a function [2]_. 
```

Comment:
The reference talks about using FFT to evaluate finite difference derivatives, which is not your use case - I am not convinced this is relevant